### PR TITLE
climbing hooks for multiz planetary maps 

### PR DESCRIPTION
**Pull request by mc-oofert**
## About The Pull Request

https://github.com/tgstation/tgstation/assets/70376633/6c65925e-5276-41fb-8062-cafc2de94b2f

allows you to climb up holes by clicking on tiles above the hole youre on
examining the rope shows you your current binds for looking up

emergency variant can be found in internals boxes on planetary maps that have multiz levels (2 uses)
a better one can be bought from cargo for 250 credits (5 uses)
syndies can buy a much much better one for 1TC or can be found in the nukie base personal lockers (10 uses)

## Why It's Good For The Game

being fucked because you fell down a 1 tile hole in the dark and now youre in a 1x1 box of snow on the middle of nowhere sucks
or falling down a hole and bam 30 watchers
or falling down a hole and now youre completely lost and might have to weld your way into the station if youre lucky

## Changelog
:cl:
add: climbing hooks that allow you to go up holes for multiz, found in internals boxes (on planetary maps), the uplink, cargo and nukie personal lockers
/:cl:

### CODE REVIEW

**Comment by Eronymun** [comment]
Instead of putting them in internals boxes I'd make them roundstart for paramedics and put them in the medical lathe. This is a really nice thing to have though!

**Comment by CoiledLamb** [comment]
put them in the mining console coward...

**Comment by mc-oofert** [comment]
> Instead of putting them in internals boxes I'd make them roundstart for paramedics and put them in the medical lathe. This is a really nice thing to have though!

this is a _selfhelp_ tool not a _get helped_ tool

**Comment by mc-oofert** [comment]
> please put them in the mining console (coward)...

which one

**Comment by Jacquerel** [comment]
I think we probably need to place some kind of "incoming" effect on the tile they are arriving at so that from the perspective of people above they don't just suddenly appear. Our existing forecast targetting circle would probably do fine.

**Comment by mc-oofert** [comment]
> I think we probably need to place some kind of "incoming" effect on the tile they are arriving at so that from the perspective of people above they don't just suddenly appear. Our existing forecast targetting circle would probably do fine.

the what now

**Comment by Jacquerel** [comment]
> > I think we probably need to place some kind of "incoming" effect on the tile they are arriving at so that from the perspective of people above they don't just suddenly appear. Our existing forecast targetting circle would probably do fine.
> 
> the what now

the graphic we dissplay for the storm staff and incoming cargo pods and probably some other stuff
I don't really care what you use but there should be _some_ visual indicator of "a guy is going to show up here" either on the destination turf or on the hole

File: code/game/objects/items/climbingrope.dm [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+	icon_state = "climbingrope"
+	inhand_icon_state = "crowbar_brass"
+	lefthand_file = 'icons/mob/inhands/equipment/tools_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/equipment/tools_righthand.dmi'
+	force = 5
+	throwforce = 10
+	reach = 2
+	throw_range = 4
+	w_class = WEIGHT_CLASS_NORMAL
+	attack_verb_continuous = list("whacks", "flails", "bludgeons", "swings")
```
Comment (Jacquerel) [inline code/game/objects/items/climbingrope.dm:14]: Wouldn't "swings" here read as `"John Mellons swings Timothy Validson with the climbing hook"`?
That doesn't sound right

**Comment by EricZilla** [comment]
God this is so good

**Pull request merged**